Cargo.toml

```diff
@@ -0,0 +1,19 @@
+[package]
+name = "near-abi"
+version = "0.1.0"
+edition = "2021"
+license = "MIT OR Apache-2.0"
+readme = "README.md"
+repository = "https://github.com/near/near-abi-rs"
+description = "NEAR smart contract ABI primitives"
+
+[dependencies]
+borsh = { version = "0.9", features = ["const-generics"] }
+serde = { version = "1", features = ["derive"] }
+schemars = "0.8.8"
+
+[dev-dependencies]
+serde_json = "1"
+
+[features]
+__chunked-entries = []
```

Comment:
Very skeptical about having a feature that's prefixed like this. Seems like the `AbiRoot::new` function should just be some function included in `__private` and not have it attached

Reply:
> Very skeptical about having a feature that's prefixed like this.

Yeah, the intent is for it to not look like something that's safe to use (hence `_` and `-`). Plus, docs.rs hides features prefixed with underscores. Thinking we should reduce the visibility as much as possible.

Reply:
sounds good, fine to keep. Had just never seen __ prefix for feature names

Reply:
yeah, me neither, got the hint from these:
 - https://github.com/rust-lang/cargo/issues/10882
 - https://github.com/rust-lang/cargo/issues/10794